Update meteor.md

xavcz opened the pull request:
Related to https://github.com/apollographql/meteor-integration/issues/77:
- don't "force" folks to use `isomorphic-fetch` by default
- specify that if you want to do server-side rendering, you should use still polyfill server-side `fetch`.

Related to `apollo-client@1.0.0-rc.0` and `apply(Batch)Middleware` (https://github.com/apollographql/meteor-integration/commit/2cf16dd768432083aab22b64b1cd86d2b30c0481#commitcomment-21102646): 
- use a `NetworkInterface` & not a `BatchingInterface` by default. A batching interface when the server actually batches queries, which happen if `dataloader` is setup for instance. As most of the user won't setup `dataloader` by default, just let's not use it by default.

cc @lorensr

lorensr commented:
We still have the first universal reason to use batching by default?

![image](https://cloud.githubusercontent.com/assets/251288/23481819/e1a8f38c-fe9a-11e6-9635-fa060eded70c.png)

https://dev-blog.apollodata.com/query-batching-in-apollo-63acfd859862#.jvinqsc21

stubailo commented on source/meteor.md:
```diff
@@ -217,14 +219,15 @@ import { ApolloProvider } from 'react-apollo';
 import { renderToStringWithData } from 'react-apollo';
 import { match, RouterContext } from 'react-router';
 import Express from 'express';
+// #1 import isomorphic-fetch so the Apollo Client can be created
```
Technically, so that the network interface can be created.

stubailo commented on source/meteor.md:
```diff
@@ -199,8 +199,10 @@ Meteor.logout(function() {
+
+1. Use `isomorphic-fetch` to polyfill `fetch` server-side (used by Apollo Client).
+2. Connect your express server to Meteor's existing server with [WebApp.connectHandlers.use](https://docs.meteor.com/packages/webapp.html)
+3. Do not end the connection with `res.send()` and `res.end()` use `req.dynamicBody` and `req.dynamicHead` instead and call `next()`. [more info](https://github.com/meteor/meteor/pull/3860)
```
Ah nice! Are there any official docs about this? Seems really useful.

stubailo commented:
I do think everyone should be setting up dataloader - you definitely can run into some pretty bad situations if you don't use it.

xavcz commented:
Thanks for your comments, it makes sense! 💯 

I've added a note about query batching to redirect to existing resources and let's encourage people to use query batching (totally agreed with "_you definitely can run into some pretty bad situations if you don't use it._" 😄)
➡️ I've updated the integration to use again the batching interface by default.

Also changed the wording in the SSR part to be more precise about network interface & `fetch` 👌 

Considering `dynamicBody/Head`, I didn't find anything official about it. I can go for a PR on the guide/docs for that: do you believe it's more suited on the [`webapp` docs](http://docs.meteor.com/packages/webapp.html)? Or on [routing section in the guide](https://guide.meteor.com/routing.html#server-side)?

helfer commented:
Thanks @xavcz!
